class CalledProcessError(Exception):
    """
    Exception raised when exception raised in child process cannot be recovered
    """